libbeat/processors/util/netinfo.go

```diff
@@ -67,6 +65,19 @@ func GetNetInfo() (ipList []string, hwList []string, err error) {
 	return ipList, unique(hwList), errs.Err()
 }
 
+// formatHardwareAddr formats hardware addresses according to the ECS spec.
+func formatHardwareAddr(addr net.HardwareAddr) string {
+	buf := make([]byte, 0, len(addr)*3-1)
```

Comment:
This will panic if passed `net.HardwareAddr{}`, can we guard against this?

Reply:
This is never called with an empty `net.HardwareAddr`.